code/modules/power/singularity/singularity.dm

```diff
@@ -511,21 +511,16 @@
 	return 1
 
 
-/obj/machinery/singularity/proc/toxmob()
-	var/toxrange = 10
-	var/toxdamage = 4
+/obj/machinery/singularity/proc/radmob()
+	var/radrange = 10
 	var/radiation = 15
 	var/radiationmin = 3
 	if(src.energy > 200)
-		toxdamage = round(((src.energy-150)/50)*4,1)
 		radiation = round(((src.energy-150)/50)*5,1)
 		radiationmin = round((radiation/5),1)
-	for(var/mob/living/M in view(toxrange, src.loc))
-		if(M.flags & INVULNERABLE)
-			continue
-		M.apply_effect(rand(radiationmin,radiation), IRRADIATE)
-		toxdamage = (toxdamage - (toxdamage*M.getarmor(null, "rad")))
-		M.apply_effect(toxdamage, TOX)
+	for(var/mob/living/M in view(radrange, src.loc))
+		radiation = rand(radiationmin, radiation)
```

Comment:
I'm pretty sure this means the first person in view will get the right level of radiation, and everyone after will get less then that.